### PR TITLE
Add tier change styling

### PR DESCRIPTION
**Pull request by ashmonty**

### CODE REVIEW

**Comment by oscie57** [comment]
imagine needing style, plain html is good enough @ashmonty 👍🏻

File: src/routers/account.js [hunk 1]
```diff
@@ -524,7 +524,8 @@ router.get('/upgrade', async (request, response) => {
 		layout: 'main',
 		locale: util.getLocale(request.locale.region, request.locale.language),
 		localeString: request.locale.toString(),
-		error: request.cookies.error
+		error: request.cookies.error,
+		currentTier: 'price_1LBnZADOJlJAaQQ3pEUjNWbY' // To be replaced
```
Comment (mrjvs) [inline src/routers/account.js:528]: cant this be implemented now that the other PR was merged?

**Review by mrjvs — CHANGES_REQUESTED** [review]

File: src/routers/account.js (hunk 1 above)
Comment (ashmonty) [reply src/routers/account.js:528]: It probably can. psst @jonbarrow

Comment (jonbarrow) [reply src/routers/account.js:528]: @ashmonty @mrjvs I will add that in with my next feature PR, I still need to add the cancellation feature

Comment (ashmonty) [reply src/routers/account.js:528]: Okie-dokie!

**Review by mrjvs — APPROVED** [review]

**Comment by ashmonty** [comment]
> imagine needing style, plain html is good enough @ashmonty 👍🏻 

the thing is I've got plenty style, and keeping it all to myself would be kinda greedy 😎

**Pull request merged**